README.md

```diff
@@ -105,7 +105,7 @@ This is usually difficult for several reasons:
     these situations with various degrees of ease - but they all
     handle them in different and incompatible ways, which again forces
     the developer to do extra work.
-  
+
```

Comment:
Empty line here. No whitespace needed even though it's indented. There's no whitespace on line 102 either.